pdns/recursordist/docs/lua-scripting/dq.rst

```diff
@@ -150,6 +150,16 @@ The DNSQuestion object contains at least the following fields:
      :param int ttl: The TTL in seconds for this record
      :param DNSName name: The name of this record, defaults to :attr:`DNSQuestion.qname`
 
+  .. method:: DNSQuestion:addRecord(type, content, place, [ttl, name])
+
+     Add a record of ``type`` with ``content`` in section ``place``.
+
+     :param int type: The type of record to add, can be ``pdns.AAAA`` etc.
+     :param str content: The content of the record, will be parsed into wireformat based on the ``type``
+     :param int place: The section to place the record, see :attr:`DNSRecord.place`
+     :param int ttl: The TTL in seconds for this record
```

Comment:
Might be good to document that the TTL defaults to 3600.

Reply:
I will do this, I just got terribly busy this week. Thanks!